### ISSUE TITLE
Finalize & publish the popover component in Figma

### ISSUE THREAD

**Issue by SeamusLeonardHPE** [grommet/hpe-design-system#4169]
Bring the figma component into parity with the new coded version of popover & prepare for publication.

**Comment by SeamusLeonardHPE** [comment]
Deploy link for guidance:
https://deploy-preview-4117--keen-mayer-a86c8b.netlify.app/templates/popover

LInk to branch 
https://www.figma.com/design/5jwtKL8dfFniymztWbvM0o/%234133---Popover?node-id=36369-2087

**Comment by KennyAtHPE** [comment]
The component has been reviewed and merged. I shared the Figma link with Brittany so that she could include it on the guidance page.

Last items
- [ ] Update guidelines and storybook link when available
- [ ] Publish and announce once Grommet component has been released and the guidance site is published (everything should be done in tandem)
- [ ] Create a follow-up ticket for [dashed-lined trigger exploration](https://www.figma.com/design/r1YUOdncOi0Bdl8msDF7Ni/Popover---%233780%2C-%233844?node-id=404-12889&t=caRohLPP0C5n5ef6-1)

**Comment by KennyAtHPE** [comment]
@SeamusLeonardHPE Opening this ticket back up as a reminder to update the storybook and guidance links whenever they are live. Tagging with blocked by internal dependencies.

Waiting on #4133 to be completed.

**Comment by SeamusLeonardHPE** [comment]
@KennyAtHPE I would suggest that these get split into separate  tasks the initial scope was only related to the figma component -

Sitting here blocked leads to lack of clarity as to what needs done and who (and what team) is responsible.


Questions - 

1. Is the figma component ready to be published or will it be impacted by any continued dev work?
2. Lets create a separate issue for documenting of this component.

**Comment by KennyAtHPE** [comment]
@SeamusLeonardHPE Removing blocked tag and closing ticket. As for your questions:
1. The component is completely ready to be published (the storybook and guidance links need to be added once those are ready on the dev side). Dev work should not impact this component since they are just fixing bugs related to Drop. I will add the publishing of the Figma component as a deliverable on the current dev ticket #4133 . 
2. Draft PR of the component is [here](https://deploy-preview-4117--keen-mayer-a86c8b.netlify.app/templates/popover). Instead, the follow-up ticket should be on exploration of the dashed-line trigger #4205.
